recipes/mongogrant/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "mongogrant" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mongogrant-{{ version }}.tar.gz
+  sha256: 2683133bb921830f71b1e0064b294491d3388095c76085028fcae0b6f3327322
+  patches:
+    - expanduser.patch
+
+build:
+  number: 0
```

Can this be noarch?
```suggestion
  noarch: python
  number: 0
```
If it is, you should add a `python >=X.Y` constraint in host and run as well

---

I explained it here https://github.com/conda-forge/staged-recipes/pull/13331#issuecomment-733940136 - it is creating a consol_script entry point so the noarch build fails.

---

See https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

---

That is what I did, but still it fails with the `consol_script` entry point https://github.com/conda-forge/staged-recipes/pull/13331/files#diff-88fc6bf745360c60ff6f0aa0a5c2a2934be5200a03e81cac4d3fd621cfd3dcd9R20

---

I guess it will work now